src/contracts/exchange/NiftyswapExchange20.sol

```diff
@@ -214,14 +258,21 @@ contract NiftyswapExchange20 is ReentrancyGuard, ERC1155MintBurn, INiftyswapExch
       // Don't need to add it for currencyReserve because the amount is added after this calculation
       uint256 currencyAmount = getSellPrice(amountSold, tokenReserve.sub(amountSold), currencyReserve);
 
-      // Increase cost of transaction
-      totalCurrency = totalCurrency.add(currencyAmount);
+      // If royalty, substract amount seller will receive after LP fees were calculated
+      // Note: Royalty will be a bit lower since LF fees are substracted first
+      (address royalty_recipient, uint256 royalty_amount) = getRoyaltyInfo(idSold, currencyAmount);
```

Comment:
minor thing, but I think you want camelCase format for variable names to be consistent with other case...?

aka.. `royaltyReceipt` and `royaltyAmount`

Reply:
Good point